README.md

```diff
@@ -68,96 +65,128 @@ you may receive this error:
+in Django admin or in a shell. It's preferable to do it in Django admin as you'll
+need to use Django admin for the next step anyway.
 
-The command lists the containers and their URLs.  Your container's URL
-will be something like ``tcp://192.168.99.100:2376``.  Micromasters will 
-run at that URL, but on port ``8079``.  So, for this example, Micromasters 
-will run at ``192.168.99.100:8079``.
+- **In Django admin**
 
-## Configuring edX devstack
+    Run the server (``paver devstack --fast lms``) and navigate to Django admin
+    (eg: http://192.168.33.10:8000/admin). In the **Authentication and Authorization**
+    section, select a **User**, or add one then select it. In the **Permissions**
+    section, check the **Superuser status** box and save.
 
-These instruction presume that you're running Micromasters in a development 
-environment that includes a local edX devstack.  If you don't have a edX
-devstack installed navigate to these
-[instructions](https://openedx.atlassian.net/wiki/display/OpenOPS/Running+Devstack)
-that walk you through the installation.  We'll wait.
+- **In a Python shell**
 
-### Create a superuser 
-   
-You need a user with superuser privileges to create the edX client for 
-Micromasters.  Start up the edX devstack and ssh into the ``edxapp`` user.
+        # Kick off an interactive shell
+        python manage.py lms --settings=devstack shell
 
-    vagrant up
-    vagrant ssh
-    sudo su edxapp
+        ### RUN THESE WITHIN THE SHELL ###
+        from django.contrib.auth.models import User
+        # We'll update the dummy user 'staff' to have superuser permissions
+        user = User.objects.get(username='staff')
+        user.is_superuser=True
+        user.save()
+
+#### 4) Add an OAuth client
+
+Run Django admin (see "In Django admin" section from step 2), navigate to the
+OAuth2 clients section (eg: http://192.168.33.10:8000/admin/oauth2/client/), and add a
+new client. Fill in the values as follows:
+
+- **User**: Use the lookup (magnifying glass) to find your superuser
+- **Name**: Anything you want. Something like 'micromasters-local'
+- **Url**: The URL where Micromasters will be running. _OSX Users: this should
+           be http://192.168.99.100:8079_
+- **Redirect uri**: Your **Url** value with "/complete/edxorg/" at the end
+
+#### 5) Copy relevant values to use in the Micromasters .env file
+
+The Micromasters codebase contains a ``.env.sample`` file which will be used as
+a template to create your ``.env`` file. For Micromasters to work, it needs 3 values:
+
+- ``EDXORG_BASE_URL``
 
-    edxapp@precise64:~/edx-platform$ pip install ipython
+    Run ``ifconfig eth1`` while ssh'ed into the machine. The
+    appropriate IP is the ``inet addr:`` value, and the LMS server runs
+    on port ``8000`` by default. The value for ``EDXORG_BASE_URL`` should be
+    something like ``http://192.168.33.10:8000``
+- ``EDXORG_CLIENT_ID`` and ``EDXORG_CLIENT_SECRET``
```

Comment:
This IP is actually [specified in the devstack VagrantFile](https://github.com/edx/configuration/blob/master/vagrant/release/devstack/Vagrantfile#L100). Unless edX changes something, it will always be ``http://192.168.33.10:8000``